pkg/okteto/kubetoken/store.go

```diff
@@ -0,0 +1,59 @@
+package kubetoken
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+func NewFileByteStore(fileName string) *FileByteStore {
+	return &FileByteStore{
+		FileName: fileName,
+		osStat:   os.Stat,
+		createFile: func(filename string) (*os.File, error) {
+			fp := filepath.Dir(filename)
+			folder := filepath.Base(filename)
+			if err := os.MkdirAll(fp, 0764); err != nil {
```

Comment:
I couldn't get these permission to `0664` because I was hitting `stat /home/az/.kubetest/okteto_auth_plugin_cache.json: permission denied 0xc0006a9e00` when the folder didn't exist